case/step/bottom_cover_kicad.py

```
@@ -0,0 +1,154 @@
+def _uid(tag):
+    return str(uuid.uuid5(_NS, tag))
+
+
+def seat_outline():
+    """The recess-opening wire (flattened frame), offset inward PLATE_MARGIN; sampled pts."""
+    pf = flatten(import_step(STEP))
+    cand = []
+    for f in pf.faces():
+        n = cm._safe_normal(f)
+        if n is not None and n.Z < SEAT_FACE_NZ and abs(f.bounding_box().min.Z - SEAT_Z_TARGET) < SEAT_Z_TOL:
+            b = f.bounding_box(); cand.append((b.size.X * b.size.Y, f))
```

Ruff (0.16.0)

[error] Multiple statements on one line (semicolon)

(E702)